src/site/stages/build/process-cms-exports/transformers/node-event.js

```diff
@@ -58,7 +58,9 @@ const transform = entity => ({
   fieldFacilityLocation: entity.fieldFacilityLocation[0] || null,
   fieldLink: createLink(entity.fieldLink, ['url']),
   fieldLocationHumanreadable: getDrupalValue(entity.fieldLocationHumanreadable),
-  fieldMedia: entity.fieldMedia[0] || null,
+  fieldMedia: entity.fieldMedia.length
```

Comment:
It's sort of picky, but `entity.fieldMedia` could feasibly be `[undefined]`, so `!!entity.fieldMedia.length === true`, but `!!entity.fieldMedia[0] === false`. If that happened, `fieldMedia` would be set to `undefined` instead of `null`.

I'm not sure if it's worth changing here and the 4 or 5 other instances, but it's a thing.